src/pubtools/sign/signers/cosignsigner.py

```diff
@@ -0,0 +1,291 @@
+from __future__ import annotations
+
+from dataclasses import field, dataclass
+import enum
+import json
+import logging
+from typing import Dict, List, ClassVar, Any
+import os
+import sys
+
+import click
+
+from . import Signer
+from ..operations.base import SignOperation
+from ..operations import ContainerSignOperation
+from ..results.signing_results import SigningResults
+from ..results import ContainerSignResult
+from ..results import SignerResults
+from ..exceptions import UnsupportedOperation
+from ..conf.conf import load_config, CONFIG_PATHS
+from ..utils import set_log_level, run_command, _get_config_file
+
+
+LOG = logging.getLogger("pubtools.sign.signers.cosignsigner")
+
+
+class SignRequestType(str, enum.Enum):
```

Comment:
Is this used anywhere?

Reply:
right, there's no need to have this for cosign signer